ext/sysvshm/sysvshm.stub.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+/** @return resource|false */
+function shm_attach(int $key, int $memsize = 10000, int $perm = 0666) {}
```

Comment:
```suggestion
function shm_attach(int $key, int $memsize = UNKNOWN, int $perm = 0666) {}
```

As this is based on the `sysvshm.init_mem` ini setting.